File: 1045.c
#include <stdio.h>
#include <math.h>

int main()
{
	double a, b, c;
	scanf("%lf %lf %lf", &a, &b, &c);
		
	double x;
	
	if(a<b){
		x = a;
		a = b;
		b = x;
	}
	if(b<c){
		x = b;
		b = c;
		c = x;
		if(a<b){
		x = a;
		a = b;
		b = x;
		}	
	}
	
	if(a >= b + c)
	{
		printf("NAO FORMA TRIANGULO\n");
	} else 
	{
	if(pow(a,2) == pow(b,2) + pow(c,2))
	{
		printf("TRIANGULO RETANGULO\n");
	} 
	
	if(pow(a,2) > pow(b,2) + pow(c,2))
	{
		printf("TRIANGULO OBTUSANGULO\n");
	}
	
	if(pow(a,2) < pow(b,2) + pow(c,2))
	{
		printf("TRIANGULO ACUTANGULO\n");
	}
	
	if((a == b) && (b == c))
	{
		printf("TRIANGULO EQUILATERO\n");
	} else

	if((a == b) || (b == c) || (a == c))
	{
		printf("TRIANGULO ISOSCELES\n");
	}
	}
	
	return 0;
}

File: python/1045.py
x  = [float(i) for i in input().split()]
l= sorted(x, reverse=True)
a = l[0]
b = l[1]
c = l[2]
if a>=(b+c):
  print("NAO FORMA TRIANGULO")
elif a<(b+c):
  if a**2 == b**2+c**2:
   print("TRIANGULO RETANGULO")
  if a**2 > b**2+c**2:
   print("TRIANGULO OBTUSANGULO")
  if a**2 < b**2+c**2:
   print("TRIANGULO ACUTANGULO")
  if a == b == c:
   print("TRIANGULO EQUILATERO")
  elif a == b or a == c or b == c:
    print("TRIANGULO ISOSCELES")